lib/install/web.rb

```diff
@@ -28,7 +28,7 @@ def apply_template!
         if ARGV.include?("--suspenders-main")
           gem "suspenders", github: "thoughtbot/suspenders", branch: "main"
         else
-          gem "suspenders"
+          gem "suspenders", "> 20240101"
```

Comment:
I _think_ I tried (and failed) to see if I could do this dynamically by including `Suspenders::Version` in this file, and then doing something like this:

```suggestion
          gem "suspenders", "> #{Suspenders::VERSION}"
```

The problem is that when this file is invoked, it's outside the context of Suspenders, so if Suspenders is not installed on the system, it will error.

Reply:
Right, this is a Ruby file executed in its own environment without access to Bundler or other dependencies. I chose `20240101` because its a lower limit on the possible values (versions starting with the `2024`-prefix), not necessarily because it's a valid version that has a release.

Reply:
> this is a Ruby file executed in its own environment without access to Bundler or other dependencies

I wonder if we could do something similar to the [Rails reproduction scripts](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#create-an-executable-test-case), but for now, this works.

Reply:
As an aside, I thought [leaving off the version meant we'd download the latest version](https://thoughtbot.com/blog/a-healthy-bundle#versionless)?

Reply:
> I wonder if we could do something similar to the [Rails reproduction scripts](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#create-an-executable-test-case)

While that might increase the runtime of `rails new`, that might be worthwhile, since it's (ideally) only executed once per project.

> As an aside, I thought [leaving off the version meant we'd download the latest version](https://thoughtbot.com/blog/a-healthy-bundle#versionless)?

Ideally, that would be true. In practice, `rails new` would consistently install `v0.2.4`.